api/CredentialsContainer.json

```diff
@@ -50,7 +50,11 @@
             "firefox": {
               "version_added": "60"
             },
-            "firefox_android": "mirror",
+            "firefox_android": {
+              "version_added": "60",
+              "partial_implementation": true,
+              "notes": "<code>credentialCreationData.attestationConveyancePreferenceOption</code> values \"direct\" and \"enterprise\" are not supported."
```

Comment:
We discussed this in the BCD call, and the preferred way to record this is to use a nested sub-feature:

`api.CredentialsContainer.create.publicKey_option.attestation`

Note that `api.CredentialsContainer.create.publicKey_option.extensions` already exists (line 116 ff.).

Reply:
@taraldr Just wanted to quickly follow-up here, and ask if you could apply the requested changes? 🙏

Reply:
It [this](https://github.com/mdn/browser-compat-data/pull/24236/commits/fa6fab1e1d86fce91b90d8b9c3f172cd207012be) more in line with how you'd prefer it?

Reply:
Thank you, I submitted a review.